docs/source/basicusage.rst

```diff
@@ -1,7 +1,10 @@
 Basic Usage
------------
+===========
 
-There are two main ways to use neo-python: ``np-prompt`` and running just the node with custom
+Introduction
+------------
+
+There are two main ways to interface with neo-python: ``np-prompt`` and running just the node with custom
```

Comment:
``np-prompt`` equals just running the node regardless of custom changes to the code. ``np-prompt`` is the same as executing `python neo/bin/prompt.py`. Having said that this addition is confusing to me.

Reply:
I edited the original line: "There are two main ways to use neo-python: np-prompt and running just the node with custom code," because I thought 'use' was confusing and interface was a better term. I added an "Introduction" header to help with the formatting for the whole document. Could you give an example of how you would recommend using/interfacing with neo-python?

Reply:
i'll withdraw my comment. Having read the full `basicusage` file I see why this is described as `interfacing in 2 ways`.